player/sub.c

```diff
@@ -81,7 +81,7 @@ void uninit_sub_all(struct MPContext *mpctx)
 }
 
 static bool update_subtitle(struct MPContext *mpctx, double video_pts,
-                            struct track *track)
+                            struct track *track, bool force_read_ahead)
```

Comment:
Is there any particular reason we need this argument and can't just use `mpctx->paused` as the value of this bool instead?